### PR TITLE
Learning rate schedule

### PR DESCRIPTION
**Pull request by Ghostvv**
**Proposed changes**:
- add possibility to have warmup-decay learning rate
- https://github.com/RasaHQ/research/issues/56

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/master/changelog) for instructions)
- [x] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

**Comment by Ghostvv** [comment]
@tabergma @dakshvar22 do you think it makes sense to add?

**Comment by tabergma** [comment]
@Ghostvv Did you run any experiments? What were the results?

**Comment by Ghostvv** [comment]
I tried on carbon-bot and Sara, didn't see consistent improvement

**Comment by dakshvar22** [comment]
@Ghostvv I would lean towards not introducing this until we see some consistent improvement or we spot some patterns where this could work. Main reason being it adds to the increasing list of hyperparams 😄

**Comment by Ghostvv** [comment]
@dakshvar22 that's was my thinking exactly

**Comment by tabergma** [comment]
@Ghostvv I think we can close this PR as we decided to not go forward with it, can't we?
